pyaedt/modules/PostProcessor.py

```diff
@@ -2824,6 +2873,106 @@ def create_fieldplot_line(self, objlist, quantityName, setup_name=None, intrinsi
             return self.field_plots[plot_name]
         return self._create_fieldplot(objlist, quantityName, setup_name, intrinsincDict, "Line", plot_name)
 
+    @pyaedt_function_handler()
+    def create_fieldplot_line_traces(
+        self,
+        seeding_faces,
+        in_volume_tracing_objs=None,
+        surface_tracing_objs=None,
+        setup_name=None,
+        intrinsincDict=None,
+        plot_name=None,
+    ):
+        """
+        Create a field plot of line.
+
+        Parameters
+        ----------
+        seeding_faces : list
+            List of seeding faces.
+        in_volume_tracing_objs : list
+            List of the in volume tracing objects.
+        surface_tracing_objs : list
+            List of the surface tracing objects.
+        setup_name : str, optional
+            Name of the setup in the format ``"setupName : sweepName"``. The default
+            is ``None``.
+        intrinsincDict : dict, optional
+            Dictionary containing all intrinsic variables. The default
+            is ``{}``.
+        plot_name : str, optional
+            Name of the fieldplot to create.
+
+        Returns
+        -------
+        type
+            Plot object.
+
+        References
+        ----------
+
+        >>> oModule.CreateFieldPlot
+        """
+        if self._app.solution_type != "Electrostatic":
+            self.logger.error("Field line traces is valid only for electrostatic solution")
+            return False
+        if intrinsincDict is None:
+            intrinsincDict = {}
+        if plot_name and plot_name in list(self.field_plots.keys()):
+            self.logger.info("Plot {} exists. returning the object.".format(plot_name))
+            return self.field_plots[plot_name]
+        if not isinstance(seeding_faces, list):
+            seeding_faces = [seeding_faces]
+        seeding_faces_ids = []
+        for face in seeding_faces:
+            if self._app.modeler[face]:
+                seeding_faces_ids.append(self._app.modeler[face].id)
+            else:
+                self.logger.error("Object {} doesn't exists in current design".format(face))
+                return False
+        if not in_volume_tracing_objs:
+            in_volume_tracing_ids = [0]
+        elif not isinstance(in_volume_tracing_objs, list):
+            in_volume_tracing_objs = [in_volume_tracing_objs]
+            in_volume_tracing_ids = []
+            for obj in in_volume_tracing_objs:
+                if self._app.modeler[obj]:
+                    in_volume_tracing_ids.append(self._app.modeler[obj].id)
+                else:
+                    self.logger.error("Object {} doesn't exists in current design".format(obj))
+                    return False
+        else:
+            self.logger.error("Provide a valid object for in volume objects selection.")
```

Comment:
```suggestion
            self.logger.error("Provide a valid object for the in-volume objects selection.")
```
Can objects be singular in the message: Provide a valid object for the in-volume object selection."